cloudbuild.yaml

```diff
@@ -17,23 +17,28 @@ steps:
     ln -s $$PWD $$shadow || stat $$shadow
 
     # Install ko from release.
-    curl -L -o ko.tar.gz https://github.com/google/ko/releases/download/v0.6.0/ko_0.6.0_Linux_i386.tar.gz
+    curl -L -o ko.tar.gz https://github.com/google/ko/releases/download/v0.8.2/ko_0.8.2_Linux_i386.tar.gz
     tar xvfz ko.tar.gz
     chmod +x ko
     alias ko=./ko
 
+    debug_tag=""
+    if [ "${TAG_NAME}" != "" ]; then
+      debug_tag=debug-${TAG_NAME}
+    fi
+
     # Use the ko binary to build the crane and gcrane builder images.
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane -t latest -t "$COMMIT_SHA"
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/gcrane -t latest -t "$COMMIT_SHA"
+    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane -t latest -t "$COMMIT_SHA" -t "$TAG_NAME"
+    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/gcrane -t latest -t "$COMMIT_SHA" -t "$TAG_NAME"
 
     # Use the ko binary to build the crane and gcrane builder *debug* images.
-    export KO_CONFIG_PATH=/workspace/.ko/debug/
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane -t debug
-    ko publish --platform=all -B github.com/google/go-containerregistry/cmd/gcrane -t debug
+    KO_CONFIG_PATH=./.ko/debug/ ko publish --platform=all -B github.com/google/go-containerregistry/cmd/crane -t debug -t "$$debug_tag"
```

Comment:
Instead of adding these new debug tags, can we instead add a new line:

```
KO_DOCKER_REPO=gcr.io/go-containerregistry/crane/debug KO_CONFIG_PATH=./.ko/debug/ ko publish --platform=all --bare github.com/google/go-containerregistry/cmd/crane -t latest -t "$COMMIT_SHA" -t "$TAG_NAME"
```
(same for gcrane)

We can keep `.../crane:debug` for backward compat, but these tags get super messy if you start having multiple dimensions within a single tag.